metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanEntityService.java

```diff
@@ -109,7 +118,7 @@ public VersionedAspect getVersionedAspect(@Nonnull Urn urn, @Nonnull String aspe
     VersionedAspect result = new VersionedAspect();
 
     if (version < 0) {
```

Comment:
maybe we move this check inside the method? so callers don't have to check if version is positive or negative?